src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -20,25 +21,33 @@
     private static final Logger logger = LogsCenter.getLogger(ModelManager.class);
 
     private final AddressBook addressBook;
+    private final FileBook fileBook;
     private final UserPrefs userPrefs;
     private final FilteredList<Person> filteredPersons;
+    private final FilteredList<EncryptedFile> filteredFiles;
 
     /**
      * Initializes a ModelManager with the given addressBook and userPrefs.
      */
-    public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs) {
+    public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyFileBook fileBook, ReadOnlyUserPrefs userPrefs) {
         super();
-        requireAllNonNull(addressBook, userPrefs);
+        requireAllNonNull(addressBook, fileBook, userPrefs);
 
-        logger.fine("Initializing with address book: " + addressBook + " and user prefs " + userPrefs);
```

Comment:
Should we keep the logger.fine for address book as well? Because when we merge we will keep have the logger for all the different new models too

Reply:
Actually I'm considering deleting this logger there will be to many books to log. But of course printing each book individually is also an option.